core/irCheck.ml

```diff
@@ -1262,6 +1328,10 @@ struct
         let tyenv = Env.bind var (info_type info) tyenv in
           (var, info), {< tyenv=tyenv >}
 
+    (* WARNING: use of remove_binder / remove_binding is only sound
```

Comment:
Should this be an issue/enhancement?

Reply:
if you want to make an issue of this then feel free... but really I just left this here for Frank's benefit

Reply:
Ah, sorry, I was reflexively complaining about commented out stuff.  Never mind.